synapse/app/phone_stats_home.py

```diff
@@ -103,10 +103,6 @@ async def phone_stats_home(hs, stats, stats_process=_stats_process):
     daily_sent_messages = await hs.get_datastore().count_daily_sent_messages()
     stats["daily_sent_messages"] = daily_sent_messages
 
-    r30_results = await hs.get_datastore().count_r30_users()
-    for name, count in r30_results.items():
-        stats["r30_users_" + name] = count
-
     r30v2_results = await hs.get_datastore().count_r30_users()
     for name, count in r30v2_results.items():
         stats["r30v2_users_" + name] = count
```

Comment:
Won't we want to rename the metric then? It feels a bit weird to me to keep a v2 around with the v1 gone. Though I don't know how much of our tooling would be confused by such a change.

Reply:
I think we'll want to keep it since you want to keep the data separate (forever).

Reply:
Indeed (what Patrick said) — old R30 data will still live on in Panopticon (and old servers that don't get upgraded will continue to send old R30). The two are not compatible, so it's much like a version number for a library, or how `/v2` endpoints in the Matrix APIs won't be renamed to `v1` again in the future just because the `v1` is obsolete (to us).

Reply:
Fair enough.